### PR TITLE
requirement #3

### PR DESCRIPTION
**Pull request by apmontgomery**

### CODE REVIEW

File: example.js [hunk 1]
```diff
@@ -1,8 +1,107 @@
...
+const app = express();
+const bodyParser = require('body-parser');
 
-app.get('/', (req, res) => res.send('Hello World!'))
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
 
-app.post('/', (req, res) => res.send('Hello Post!'))
+
+let events = [
```
Comment (gootyfer) [inline example.js]: Variable never re-assigned: better use `const`

File: example.js [hunk 2]
```diff
@@ -1,8 +1,107 @@
...
+  },
+  {
+    id: "2",
+    title: "The second coding Session",
+    description: "this is the second description",
+    date: new Date("2017-03-29T00:00:00Z")
+  }
+]
+
+id = 2;
```
Comment (gootyfer) [inline example.js]: Use `let` in this variable declaration

File: example.js [hunk 3]
```diff
@@ -1,8 +1,107 @@
...
+
+app.get('/events/:id/', (req, res) => {
+  findEventIndexById(req.params.id) > -1 ? res.json(events[findEventIndexById(req.params.id)]) : res.status(404).json({message: "No event found with that ID"})
+})
+
+app.post('/events/', (req, res) => {
+  let event = req.body;
+  id++;
+  event.id = id + '';
+  event.date = new Date(event.date);
```
Comment (gootyfer) [inline example.js]: You can also validate the date data in case it was not OK

File: example.js (hunk 1 above)
Comment (apmontgomery) [inline example.js]: thanks. made that change

File: example.js (hunk 2 above)
Comment (apmontgomery) [inline example.js]: thank you

File: example.js (hunk 3 above)
Comment (apmontgomery) [inline example.js]: are you suggesting adding a validation on top of new Date or replacing that?

Comment (gootyfer) [inline example.js]: @apmontgomery imagine the `event.date` is not a valid format. Then, `new Date()` will return a non-valid date. Check that and maybe answer the request with an error.

**Pull request merged**